src/Umbraco.Cms.Integrations.Crm.Dynamics/Api/Management/Controllers/GetAccessTokenController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Cms.Integrations.Crm.Dynamics.Configuration;
+using Umbraco.Cms.Integrations.Crm.Dynamics.Models.Dtos;
+using Umbraco.Cms.Integrations.Crm.Dynamics.Services;
+
+namespace Umbraco.Cms.Integrations.Crm.Dynamics.Api.Management.Controllers
+{
+    public class GetAccessTokenController : FormsControllerBase
+    {
+        public GetAccessTokenController(IOptions<DynamicsSettings> options, DynamicsService dynamicsService, DynamicsConfigurationService dynamicsConfigurationService, DynamicsComposer.AuthorizationImplementationFactory authorizationImplementationFactory) : base(options, dynamicsService, dynamicsConfigurationService, authorizationImplementationFactory)
+        {
+        }
+
+        [HttpPost("get-access-token")]
```

Comment:
I recall Adrian made some suggestions and changes to align the routes defined on the Shopify management API - and for this one suggestion was to remove the `get-` prefix here (so it became [this](https://github.com/umbraco/Umbraco.Cms.Integrations/blob/d43cffdd74fd6fe904fcf928b11e009f62001ed3/src/Umbraco.Cms.Integrations.Commerce.Shopify/Api/Management/Controllers/GetAccessTokenController.cs#L19)).  So can you do that please, and also compare to the other integrations we have to try to make sure all are as similar as they can be?